### PR TITLE
Add documentation to rabbitmq bindings for Priority Queue

### PR DESCRIPTION
**Pull request by kzmake**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Documentation to RabbitMQ binging for Priority Queue
- `maxPriority` param for queue
- `priority` param for message

## Issue reference

- https://github.com/dapr/components-contrib/pull/606

Closes: #1101

### CODE REVIEW

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md [hunk 1]
```diff
@@ -69,6 +72,30 @@ curl -X POST http://localhost:3500/v1.0/bindings/myRabbitMQ \
...
+
+```shell
+curl -X POST http://localhost:3500/v1.0/bindings/myRabbitMQ \
+  -H "Content-Type: application/json" \
+  -d '{
+        "data": {
+          "message": "Hi"
+        },
+        "metadata": {
+          "priority": "5"
```
Comment (msfussell) [inline daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md:93]: The priority field is not described in the metadata parameters above (only maxPriority is). Can you add priority to this also, with a description of how/when to use this? 

Also what the relationship between setting priority and maxPriority on the binding?

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md [hunk 2]
```diff
@@ -40,6 +42,7 @@ spec:
 - `ttlInSeconds` is an optional parameter to set the [default message time to live at RabbitMQ queue level](https://www.rabbitmq.com/ttl.html). If this parameter is omitted, messages won't expire, continuing to exist on the queue until processed.
 - `prefetchCount` is an optional parameter to set the [Channel Prefetch Setting (QoS)](https://www.rabbitmq.com/confirms.html#channel-qos-prefetch). If this parameter is omiited, QoS would set value to 0 as no limit.
 - `exclusive` determines whether the topic will be an exclusive topic or not
+- `maxPriority` is an optional parameter to set the [priority queue](https://www.rabbitmq.com/priority.html). If this parameter is omitted, queue will be created as a general queue instead of a priority queue.
```
Comment (msfussell) [inline daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md:45]: If would be useful to give an example here. For example what does setting this to "5" mean? Is this the priority level of the message, which when set with the priority parameter is treated as a high priority message?

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md (hunk 1 above)
Comment (yaron2) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md:93]: The priority field is correctly shown here as metadata you give the API. It's for the publisher to set the priority of the message and it doesn't appear on the RabbitMQ component.

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md (hunk 2 above)
Comment (yaron2) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md:45]: Yes, it's sets the maximum priority on the queue. it's further explained in the URL linked here.

**Review by AaronCrawfis — APPROVED** [review]

**Comment by kzmake** [comment]
@yaron2 Thank you for your support!

**Pull request merged**